code/game/area/areas/centcom.dm

```diff
@@ -113,6 +113,12 @@
 	name = "Thunderdome (Observer.)"
 	icon_state = "purple"
 
+/area/fabric_of_reality
+	name = "Tear in the Fabric of Reality"
```

Comment:
Pretty sure most shit in SS13 item names is supposed to be uncapitalised. Don't think I have any other complaints though. I saw it in action and it's a nifty PR.

Reply:
this is an area name, not an item name

Reply:
![image](https://user-images.githubusercontent.com/54416597/174482741-df3d7b19-f9f4-4fce-90f4-0c67687325e0.png)
they're all capitalized

Reply:
i am blind